### ISSUE TITLE
4.12.x compatibility 

### ISSUE THREAD

**Issue by jankout** [contao/tcpdf-bundle#3]
Hello, 

unfortunately, I cannot install your extension with Contao 4.12.x. I got this error message. 

```
contao/tcpdf-bundle[1.2.0, ..., 1.2.1] require symfony/dependency-injection 3.4.* || ^4.1 -> satisfiable by symfony/dependency-injection[v3.4.0, ..., v3.4.47, v4.1.0, ..., v4.4.27].
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.4.27.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.4.26.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.4.25.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.4.24.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.4.23.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.4.22.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.4.21.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.4.20.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.4.19.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.4.18.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.4.17.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.4.16.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.4.15.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.4.14.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.4.13.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.4.12.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.4.11.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.4.10.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.4.9.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.4.8.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.4.7.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.4.6.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.4.5.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.4.4.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.4.3.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.4.2.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.4.1.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.4.0.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.3.11.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.3.10.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.3.9.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.3.8.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.3.7.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.3.6.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.3.5.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.3.4.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.3.3.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.3.2.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.3.1.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.3.0.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.2.12.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.2.11.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.2.10.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.2.9.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.2.8.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.2.7.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.2.6.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.2.5.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.2.4.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.2.3.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.2.2.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.2.1.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.2.0.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.1.12.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.1.11.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.1.10.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.1.9.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.1.8.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.1.7.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.1.6.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.1.5.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.1.4.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.1.3.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.1.2.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.1.1.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v4.1.0.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.47.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.46.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.45.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.44.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.43.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.42.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.41.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.40.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.39.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.38.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.37.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.36.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.35.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.34.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.33.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.32.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.31.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.30.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.29.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.28.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.27.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.26.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.25.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.24.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.23.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.22.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.21.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.20.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.19.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.18.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.17.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.16.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.15.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.14.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.13.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.12.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.11.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.10.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.9.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.8.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.7.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.6.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.5.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.4.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.3.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.2.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.1.
    - symfony/web-profiler-bundle v5.3.5 conflicts with symfony/dependency-injection v3.4.0.
    - contao/manager-bundle 4.12.1 requires symfony/web-profiler-bundle ^4.4 || ^5.2 -> satisfiable by symfony/web-profiler-bundle[v5.3.5].
```

**Comment by jankout** [comment]
I could install it at least, but only with Symfony 4. Could you make it compatible with Symfony 5, please?

**Comment by leofeyer** [comment]
Fixed in cc591aaf207953cb0bafca0cd490c089cfd90dda.
